spec/iron_bank/actions/update_spec.rb

```diff
@@ -7,7 +7,7 @@
   it_behaves_like 'a Zuora action' do
     let(:args) do
       {
-        type:    'Account',
+        type:    :account,
```

Comment:
Something is going on here. `Account` is the accepted Zuora's **value**, it is not a field. I don't think we should be passing in `:account` here -**or**- we should be accepting both.

Reply:
It will accept both `:account` & `'Account'` (as they both camelize to the same string.)